crates/ruff_dev/src/generate_docs.rs

```diff
@@ -25,7 +25,7 @@ pub(crate) fn main(args: &Args) -> Result<()> {
     for rule in Rule::iter() {
         if let Some(explanation) = rule.explanation() {
             let mut output = String::new();
-            output.push_str(&format!("# {} ({})", rule.as_ref(), rule.noqa_code()));
+            output.push_str(&format!("# (`{}`) {}", rule.noqa_code(), rule.as_ref()));
```

Comment:
```suggestion
            output.push_str(&format!("# ({}) {}", rule.noqa_code(), rule.as_ref()));
```
To fix the browser tab title showing the backticks:
![image](https://github.com/astral-sh/ruff/assets/19514231/951242ec-4bef-44de-887b-032ab993c8d4)
I'm not sure which would be preferable, having the code block or fixing the tab title.